src/addons/dragAndDrop/withDragAndDrop.js

```diff
@@ -52,8 +52,9 @@ try {
  * If you care about these corner cases, you can examine the `allDay` param suppled
  * in the callback to determine how the user dropped or resized the event.
  *
- * Note: you cannot use custom `EventWrapper`, `DayWrapper` or `DateCellWrapper`
- * components when using this HOC as they are overwritten here.
+ * Note: you cannot use custom `EventWrapper` components when using this HOC as it
```

Comment:
@ehahn9 I think it was you who did this initial implementation. Did you try implementing custom `EventWrapper` support?

Reply:
Thanks for the ping... Indeed I took a look at it but wimped out due to lack of time. This was on my wish list (*), so thanks for working on it! Our app doesn't require it, so I probably won't have time to review your excellent PR, tho. Anyone else?

(*) my other two hot buttons were per-event draggability and much better handling to the click geometry for dnd (right now the drag tracking is truly funky because it isn't based on the mousedown location.. others had spent time on the initial DnD implementation but we all gave up...)